client/python/modeldb/__main__.py

```diff
@@ -0,0 +1,182 @@
+import os
+import json
+
+
+def to_bool(value):
+    valid = {'true': True, 't': True, '1': True,
+             'false': False, 'f': False, '0': False,
+             }
+
+    if isinstance(value, bool):
+        return value
+
+    if not isinstance(value, basestring):
+        raise ValueError('invalid literal for boolean. Not a string.')
+
+    lower_value = value.lower()
+    if lower_value in valid:
+        return valid[lower_value]
+    else:
+        raise ValueError('invalid literal for boolean: "%s"' % value)
+
+
+if __name__ == '__main__':
```

Comment:
Nit: Can we have this helper script be called modeldb_config or something? python -m modeldb makes it seem that we are running modeldb.